src/mame/drivers/twincobr.cpp

```diff
@@ -1415,13 +1397,21 @@ void twincobr_state::init_twincobr()
 }
 
 
+void twincobr_state::init_fsharkbt()
+{
+	init_twincobr();
+	m_gxl->dsp().space(AS_IO).install_readwrite_handler(0x02, 0x02,
+		read16smo_delegate(*this, FUNC(twincobr_state::fsharkbt_dsp_r)), write16smo_delegate(*this, FUNC(twincobr_state::fsharkbt_dsp_w)), 0xffff);
+}
```

Comment:
I really don’t like the idea of a change that requires adding init functions for installing handlers.  That seems regressive.